Dockerfile

```diff
@@ -25,14 +25,38 @@ ENV CONSUL_ADDRESS="" \
     USERS="" \
     EXTRA_FRONTEND=""
 
+# Merge telegraf to this so we can extract stats
+
+ENV TELEGRAF_VERSION 1.1.1
```

Comment:
I'm not sure how to grab metrics from the container when it runs as a service. I put this in because that was the fastest way to demonstrate the solution I was working on. If I could instead expose the stats on a unix socket and expose that via host shared volume, it might work. I shall remove this - since it's not useful unless you plan to use it exactly like I did :)